Library/Formula/extract_url.rb

```diff
@@ -0,0 +1,23 @@
+class ExtractUrl < Formula
+  desc "Perl script to extracts URLs from emails or plain text."
+  homepage "http://www.memoryhole.net/~kyle/extract_url/"
+  url "https://extracturl.googlecode.com/files/extract_url-1.5.8.tar.gz"
+  sha256 "58eac907cb926deba74ab81e7503a1055fd3cbe20952f011d8e6b75da12d6bcc"
+
+  depends_on "MIME::Parser" => :perl
```

Comment:
We'd prefer to vendor these rather than ordering the user to install them before they can install the formula. The latter tends to be bad UX.